File: Java/Lista_04(Vetores)/Numeros_Pares.java
import java.util.Scanner;

public class Numeros_Pares {

    public static void main(String[] args) {

        Scanner s = new Scanner(System.in);

        System.out.print("Quantos numeros voce vai digitar? ");
        int n = s.nextInt();

        int vet[] = new int[n];

        for (int i = 0; i < vet.length; i++) {
            System.out.print("Digite um numero: ");
            vet[i] = s.nextInt();
        }

        System.out.println("\nNUMEROS PARES:");

        int cont = 0;

        for (int i = 0; i < vet.length; i++) {
            if (vet[i] % 2 == 0) {
                System.out.print(vet[i] + " ");
                cont++;
            }
        }

        System.out.print("\n\nQUANTIDADE DE PARES: " + cont);

        s.close();

    }
}

File: Python/Lista_04(Vetores)/Numeros_Pares.py
n = int(input("Quantos numeros voce vai digitar? "))

vet = [0 for x in range(n)]

for i in range(0,n):
    vet[i] = int(input("Digite um numero: "))

print("\nNUMEROS PARES:")

for i in range(0,n):
    if vet[i]%2==0:
        print(vet[i], end="  ")

cont = 0

for i in range(0,n):
    if vet[i]%2==0:
        cont+=1

print(f"\n\nQUANTIDADE DE PARES = {cont}")